tests/test_sklearn_linear_converter.py

```diff
@@ -202,7 +202,7 @@ def test_sgd_classifier_multi(self):
 
     # SGDClassifier with modified huber loss
     @unittest.skipIf(
-        LooseVersion(torch.__version__) < LooseVersion("1.6.0"), reason="Modified Huber loss test requires torch >= 1.6.0"
```

Comment:
Wait can we just remove this? Now we only support Pytorch >= 1.6.0.

Reply:
This one is ok to remove because we are only supporting >=1.6, so it doesn't apply to < 1.6.